module/VuFind/src/VuFind/GeoFeatures/BasemapConfig.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Basemap Configuration Module
+ *
+ * PHP version 5
+ *
+ * Copyright (C) Villanova University 2010.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  GeoFeatures
+ * @author   Leila Gonzales <lmg@agiweb.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org/wiki/vufind2:recommendation_modules Wiki
+ */
+namespace VuFind\GeoFeatures;
+
+use Zend\ServiceManager\ServiceManager;
+
+/**
+ * Basemap Configuration Class
+ *
+ * @category VuFind
+ * @package  GeoFeatures
+ * @author   Leila Gonzales <lmg@agiweb.org>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:hierarchy_components Wiki
+ */
+class BasemapConfig
+{
+    /**
+     * Basemap tileserver URL
+     * Default is the wikimedia osm-intl map
+     *
+     * @var string
+     */
+    protected $basemap_url = "https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png";
```

Comment:
For style consistency, I'd suggest changing this to basemapUrl (I realize we do have some underscores in property names elsewhere, but it's generally frowned upon).